api/app.js

```diff
@@ -2,10 +2,11 @@ var express = require('express');
 var bodyParser = require('body-parser');
 var config = require('./controllers/config');
 var authController = require('./controllers/auth');
+var groupAdminController = require ('./controllers/groupAdmin');
```

Comment:
Use const\let, not var. I already added rule for it in eslint, but I want to point it out.